### ISSUE TITLE
[BITV] 9.1.3.1h/8.1 - When editing a user in the "table", input fields appear in the cells that can be used to edit the data. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36922]
The input field in the "Username" column does not have an accessible name, and the other fields only use placeholder attributes, which may not be read depending on the browser and assistive technology. The problem is further exacerbated by the fact that the table was not implemented as such, i.e. there is no connection to the column headers. 

- [ ] Fix
- [ ] Backport

![4b98244143224a248f230b7ff75c24c8](https://user-images.githubusercontent.com/1315170/221936300-ee818269-b687-4cd2-9ec6-b22f2f445086.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-fa5c8d46f9-v8-n1

</details>
